.claude/skills/twelve-factor-docker/scripts/agents-runner.sh

```
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Agent runner for twelve-factor-docker
+set -euo pipefail
+
+ROOT="$(git rev-parse --show-toplevel)"
+SKILL_DIR="$ROOT/.claude/skills/twelve-factor-docker"
+AGENTS_DIR="$SKILL_DIR/agents"
+
+source "$SKILL_DIR/scripts/bus.sh"
+
+MODE="${1:-run}"
+MAX_AGENTS="${AGENTS_MAX:-4}"
```

Shellcheck (0.11.0)

[warning] MAX_AGENTS appears unused. Verify use (or export if used externally).

(SC2034)